_dashboards/index.md

```diff
@@ -11,15 +11,18 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# What is OpenSearch Dashboards?
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is an open-source analytics and search application for the OpenSearch engine. With OpenSearch Dashboards, you can query, visualize, alert on, and understand your metrics no matter where they are stored. 
```

Comment:
The "no matter where they are stored" may be misleading. At the moment, the data will need to be stored in their OpenSearch cluster. In the future, this will be different.

Reply:
Rewrote section